File: app/src/main/java/com/xianzhitech/ptt/engine/TalkEngine.java
package com.xianzhitech.ptt.engine;


import android.support.annotation.NonNull;

import com.xianzhitech.ptt.model.Room;

/**
 *
 * 定义对讲引擎操作的接口
 *
 * Created by fanchao on 13/12/15.
 */
public interface TalkEngine {
    void connect(@NonNull  Room room);
    void dispose();
    void startSend();
    void stopSend();
}

File: app/src/main/kotlin/com/xianzhitech/ptt/engine/TalkEngine.kt
package com.xianzhitech.ptt.engine


/**

 * 定义对讲引擎操作的接口

 * Created by fanchao on 13/12/15.
 */
interface TalkEngine {
    fun connect(roomId: String, property: Map<String, Any?>)
    fun dispose()
    fun startSend()
    fun stopSend()
}
